### PR TITLE
Remove all dangling PRs and branches from auto deps update

### PR DESCRIPTION
**Pull request by chxchx**
**Release note**:

```release-note
none
```

### CODE REVIEW

**Comment by chxchx** [comment]
I signed it!

File: toolbox/util/githubClient.go [hunk 1]
```diff
@@ -341,11 +346,11 @@ func (g GithubClient) CreateReleaseUploadArchives(repo, releaseTag, archiveDir s
 	}
 	release.Draft = newBool(true)
 	release.Prerelease = newBool(true)
-	if releaseBody, err := FillUpTemplate(releaseBodyTemplate, map[string]string{"ReleaseTag": releaseTag}); err != nil {
+	releaseBody, err := FillUpTemplate(releaseBodyTemplate, map[string]string{"ReleaseTag": releaseTag})
```
Comment (chxchx) [inline toolbox/util/githubClient.go:349]: This is only cosmetic. My local linter was yelling.

**Review by sebastienvas — APPROVED** [review]

**Pull request merged**